### PR TITLE
Swagger/updates/2022.10.28

### PR DESCRIPTION
**Pull request by snissn**
documentation fixes for swagger comments to support postman and docusaurus plugins

### CODE REVIEW

File: pinning.go [hunk 1]
```diff
@@ -693,15 +696,18 @@ func (s *Server) handleAddPin(e echo.Context, u *util.User) error {
 	if err != nil {
 		return err
 	}
-	return e.JSON(http.StatusAccepted, status)
+	return e.JSON(http.StatusOK, status)
```
Comment (en0ma) [inline pinning.go:699]: @snissn the IPFS spec expects a 202 and not a 200 - https://ipfs.github.io/pinning-services-api-spec/#operation/addPin

Comment (snissn) [reply pinning.go:699]: Thank you! I was confused by the 202, i'll update this. Thanks for catching!

**Comment by snissn** [comment]
@en0ma  [e561cbf](https://github.com/application-research/estuary/pull/501/commits/e561cbfa6b0f5d8a0aeaf8faf7f11f3380d2e21b)

**Review by en0ma — APPROVED** [review]
LGTM

**Review by jlogelin — APPROVED** [review]
LGTM

**Review by alvin-reyes — APPROVED** [review]
LGTM

**Comment by snissn** [comment]
ty! you guys rock!

giphy

**Pull request merged**